test/tst_distortedview.jl

```diff
@@ -116,16 +116,16 @@ end
     @test parent(dv) === camera
     @test size(dv) == size(camera)
     @test eltype(dv) == eltype(camera)
-    @test summary(dv) == "512×512 Augmentor.DistortedView(::Array{Gray{N0f8},2}, ::Array{Float64,3} as 3×3 vector field) with element type ColorTypes.Gray{FixedPointNumbers.Normed{UInt8,8}}"
+    @test summary(dv) == "512×512 Augmentor.DistortedView(::Array{Gray{N0f8},2}, ::Array{Float64,3} as 3×3 vector field) with eltype Gray{Normed{UInt8,8}}"
     @test_reference "reference/distort_static.txt" dv
 
     camerao = OffsetArray(camera, (-5,-10))
     dv2 = @inferred Augmentor.DistortedView(camerao, A)
     @test size(dv2) == size(camera)
     @test eltype(dv2) == eltype(camera)
-    @test summary(dv2) == "512×512 Augmentor.DistortedView(::OffsetArray{Gray{N0f8},2}, ::Array{Float64,3} as 3×3 vector field) with element type ColorTypes.Gray{FixedPointNumbers.Normed{UInt8,8}}"
+    @test summary(dv2) == "512×512 Augmentor.DistortedView(::Array{Gray{N0f8},2}, ::Array{Float64,3} as 3×3 vector field) with eltype Gray{Normed{UInt8,8}}"
```

Comment:
What happened here?

Reply:
Looks like it's related to my previous added specification to `DistortedView`

It's a WIP commit, I'll recheck that later when I fix julia 1.3 deprecations.

Reply:
Rechecked in https://github.com/Evizero/Augmentor.jl/commit/733e32bebf8176bc0843b9679029d3ae147047b9

To successfully compare two `DistortedView`s `A` and `B`, their axes should be the same, i.e., `axes(A)==axes(B)`, so I add the `plain_axes` in the construction method to make sure this. This is still lazy-mode since `plain_axes` doesn't copy data. The test in `tst_distortion.jl` checks that as well.